packages/opentelemetry/test/unit/exporter-specs.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-unused-vars */
```

Comment:
do we need to disable this check?

Reply:
same below

Reply:
we need it, linter complains for the types imported for docstrings

Reply:
I would prefer then to only apply it to the particular lines. this rule could still be useful to find potential unused vars in order places of the module

Reply:
also there are no docstrings in tests, what linter is complaining about there?